### PR TITLE
improve fuzzed sprite visibility in hires mode

### PR DESCRIPTION
**Pull request by fabiangreffrath**

### CODE REVIEW

**Comment by fabiangreffrath** [comment]
Idea is to put this into a separate function and provide a choice in the menu, e.g. "original", "blocky" and "translucent" (which would replace `vis->colormap = NULL` with `vis->mobjflags |= MF_TRANSLUCENT` in `R_ProjectSprite()`).

**Comment by fabiangreffrath** [comment]
I need some feedback to this, please. :wink: It is currently meant to only apply to hires mode, of course.

**Comment by rfomin** [comment]
Interesting feature, I like it. Maybe blocky specters are too easy to spot now, but I think we should do it.

**Comment by fabiangreffrath** [comment]
This needs one or two more comments to critical code changes, but should be ready otherwise.

File: Source/r_draw.c [hunk 1]
```diff
@@ -385,6 +385,78 @@ void R_DrawFuzzColumn(void)
...
+int fuzzcolumn_mode = 0;
+void (*R_DrawFuzzColumn) (void) = R_DrawFuzzColumn_orig;
+void R_SetFuzzColumnMode (void)
+{
+  if (fuzzcolumn_mode == 2)
+    R_DrawFuzzColumn = R_DrawTLColumn;
+  else if (fuzzcolumn_mode == 1)
+    R_DrawFuzzColumn = hires ? R_DrawFuzzColumn_block : (fuzzcolumn_mode = 0, R_DrawFuzzColumn_orig);
+  else
+    R_DrawFuzzColumn = R_DrawFuzzColumn_orig;
```
Comment (rfomin) [inline Source/r_draw.c:457]: I've got an access violation here. Backtrace:
```
00000000`0029f7a0 00000001`3f5daf7b woof!R_SetFuzzColumnMode+0x67 [C:\develop\woof\Source\r_draw.c @ 457]
00000000`0029f7c0 00000001`3f54352a woof!R_ExecuteSetViewSize+0x45b [C:\develop\woof\Source\r_main.c @ 471]
00000000`0029f820 00000001`3f5431f1 woof!D_Display+0x2a [C:\develop\woof\Source\d_main.c @ 239]
00000000`0029f8a0 00000001`3f562407 woof!D_DoomMain+0x11b1 [C:\develop\woof\Source\d_main.c @ 2384]
00000000`0029f970 00000001`3f61f042 woof!SDL_main+0x87 [C:\develop\woof\Source\i_main.c @ 113]
```

Comment (rfomin) [reply Source/r_draw.c:466]: Works now, after full rebuild.

**Comment by rfomin** [comment]
Looks good to me, but "translucent mode" with default settings isn't very transparent at all:
doom01
"blocky mode":
doom03

**Comment by fabiangreffrath** [comment]
Hm, it's probably a bit too easy that way, right? Should we drop "translucent" drawing mode and just leave "original" and "blocky"?

**Comment by rfomin** [comment]
> Should we drop "translucent" drawing mode and just leave "original" and "blocky"?

Yes, I think so.

**Review by rfomin — APPROVED** [review]

**Pull request merged**